.github/workflows/push-xc-admin-frontend-image.yml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches: [main]
     paths: [governance/xc_admin/**]
+  pull_request:
+    paths: [governance/xc_admin/**]
```

Comment:
Note that the push below is already gated on the main branch

Reply:
got it, I was just checking on that